app/Http/Controllers/LoginCredentialController.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\{
+    User,
+    LoginCredential
+};
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Str;
+use App\Http\Requests\LoginCredentialRequest;
+
+class LoginCredentialController extends Controller
+{
+    public function create(Request $request)
+    {
+        return view('login_credential.create');
+    }
+
+    public function store(LoginCredentialRequest $request)
+    {
+        $user = User::where('email', $request->email)->first();
+        if(!is_null($user) && !Hash::check($request->password,$user->password)){
+            return to_route('login_credential.create')->withErrors(['login_error' => 'パスワードが一致しません']);
+        }
```

Comment:
```suggestion
        if(!is_null($user) && !Hash::check($request->password,$user->password)){
            return to_route('login_credential.create')->withErrors(['login_error' => 'パスワードが一致しません']);
        }
```
form requestで同じ処理があるので、消してください。

Reply:
コントローラー内の記述を削除し、formリクエストも修正致しました。
ご確認宜しくお願い致します。

app/Http/Requests/LoginCredentialRequest.php
```
                function ($attribute, $value, $fail) {
                    if (is_null(User::where('email', $this->email)->first()) || !Hash::check($value, User::where('email', $this->email)->value('password'))) {
                        $fail(':attributeが一致していません。');
                    }
                }
```
app/Http/Controllers/LoginCredentialController.php